model/task/task.go

```diff
@@ -663,7 +665,9 @@ func (t *Task) isSystemUnresponsive() bool {
 }
 
 func (t *Task) SetOverrideDependencies(userID string) error {
+	dependenciesMetTime := time.Now()
 	t.OverrideDependencies = true
+	t.DependenciesMetTime = dependenciesMetTime
```

Comment:
Small nit: do `t.DependenciesMetTime = time.Now()` and then in the query below reference it like the cache `DependenciesMetTimeKey: t.DependenciesMetTime`